demos/optimization.py

```diff
@@ -30,7 +30,7 @@ def next(self):
     cerebro = bt.Cerebro(maxcpus=1)
 
     data = bt.feeds.YahooFinanceCSVData(
-        dataname="datas/orcl-1995-2014.txt",
+        dataname="demos/datas/orcl-1995-2014.txt",

```

Comment:
When running locally on vscode the filepath that YahooFinanceCSVData() used was the root directory instead of the demos directory though, and that's why I had to change this to find the file. Is that something of concern?

Another thing is that while running locally the edits in bokeh_webapp.py don't show up - I tried commenting out the print(f"Open your browser here: http://localhost:{port}") statement but it didn't change the output. At first I thought it might be a module import caching issue but couldn't fix with reload/refreshing. Any idea what's going on? Thanks!

Reply:
1.) I don't use vscode but I imagine you have to set the working directory of your run configuration to the `demos` directory. I guess otherwise it will default to the project's root directory.

2.) I would try stopping in the code in th debugger. You should be able to see the full file path it actually is running on. Maybe you installed the package and you are running the code from the regular `site-packages` directory instead of your checkout directory?